common/scala/src/main/scala/whisk/core/entity/ActivationId.scala

```diff
@@ -88,13 +91,23 @@ protected[core] object ActivationId extends ArgNormalizer[ActivationId] {
         def read(value: JsValue) = Try {
             val JsString(s) = value
```

Comment:
Can the id ever contain only numbers, be decoded into `JsNumber`, and have this throw a `MatchError`?